### PR TITLE
[4.0] Correct minimum version check for MariaDB on installation

### PR DESCRIPTION
**Pull request by richard67**
Pull Request for Issue [https://github.com/joomla/joomla-cms/pull/25285#issuecomment-504428587](https://github.com/joomla/joomla-cms/pull/25285#issuecomment-504428587) .

### Summary of Changes

- Add new UI text `INSTL_DATABASE_INVALID_MARIADB_VERSION` to installation language files.
- When check for minimum DB version fails with mysqli or mysql driver, check if MariaDB and if so, use the right UI text for the exception.

### Testing Instructions

1. Try to install current 4.0-dev on a MariaDB of a version which does not fulfill the minimum reguirement 10.0.0. Result: See section "Actual result" below.
2. Apply the changes of this PR and try again. Result: See section "Expected result" below.

### Expected result

Error message like `You need MariaDB 10.0 or higher to continue the installation. Your version is: 5.5.60-MariaDB`.

### Actual result

Error message like `You need MySQL 10.0 or higher to continue the installation. Your version is: 5.5.60-MariaDB`.

### Documentation Changes Required

None.

### CODE REVIEW

File: installation/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -178,16 +178,17 @@ INSTL_DATABASE_ERROR_READING_SQL_FILE="Could not read SQL file."
 INSTL_DATABASE_FIELD_VALUE_REMOVE="Remove"
 INSTL_DATABASE_FIELD_VALUE_BACKUP="Backup"
 INSTL_DATABASE_FIX_LOWERCASE="The table prefix must be lowercase for PostgreSQL."
-INSTL_DATABASE_FIX_TOO_LONG="The MySQL table prefix must be a maximum of 15 characters."
+INSTL_DATABASE_FIX_TOO_LONG="The table prefix must be a maximum of 15 characters for MySQL or MariaDB."
```
Comment (wilsonge) [inline installation/language/en-GB/en-GB.ini]: Leave these strings as MySQL - given mariadb is a mysql replacement no need to make them more confusing. Just need the new invalid mariadb version string

**Comment by richard67** [comment]
@wilsonge Done.

**Comment by wilsonge** [comment]
Thanks!

**Comment by richard67** [comment]
Thanks, too.

**Pull request merged**